index.html

```diff
@@ -10,22 +10,23 @@
         <title>Friday</title>
     </head>
     <body>
-        <header>
-            <div class="header-container">
+        <header id="header-bg">
+            <div class="header-container" id="header">
                 <nav class="navigation">
                     <ul>
-                        <li><a><img src="images/image_logo.png">Friday</a></li>
-                        <li><a>About</a></li>
-                        <li><a>Contact</a></li>
-                        <li><a><img src="#"></a></li>
+                        <!-- Need help with alignment, please. -->
```

Comment:
hi Daniel, you could set the ul as a flex by display: flex, so as to align them according to screen sizes